pom.xml

```diff
@@ -335,6 +336,11 @@
           <version>2.17</version>
           <dependencies>
             <dependency>
+              <groupId>org.apache.apex</groupId>
+              <artifactId>apex-codestyle-config</artifactId>
+              <version>${project.version}</version>
```

Comment:
This can be resolved by giving the artifact a fixed, separate version number. 1.0.0?